swan-lake/resources/featured-scenarios/create-your-first-connector-with-ballerina.md

```diff
@@ -0,0 +1,169 @@
+
+1. In your terminal, run the following command from the project root:
+   ```bash
+   bal openapi -i path/to/spec --mode client -o ballerina
+   ```
+
+   Example for Twitter:
+   ```bash
+   bal openapi -i docs/spec/openapi.json --mode client -o ballerina
+   ```
+
+   This will generate the connector code in your `ballerina` directory.
+
+2. Make sure that the generated client consists of the following files:
+   - `client.bal`: Contains the client implementation with all the API operations.
+   - `types.bal`: Contains the data types used in the client.
+   - `utils.bal`: Contains utility functions used in the client.
+
+## Step 4: Test the connector
+
+Now that your connector is generated, it’s important to write tests to ensure everything works as expected.
+
+1. In the `ballerina/tests` directory, add the required test files and write test cases for the key operations of your connector. Aim to cover as many API use cases as possible.
+
+2. Execute the tests using the `bal test` command to verify the functionality of your connector.
+
+> For detailed guidance on writing tests, check the [Ballerina testing guide](https://ballerina.io/learn/test-ballerina-code/test-a-simple-function/).
+
+> For sample implementations, refer to the [test implementation of the Ballerina Twitter connector](https://github.com/ballerina-platform/module-ballerinax-twitter/tree/main/ballerina/tests).
+
+## Step 5: Document the Connector
+
+Follow these steps to ensure your connector is well-documented:
+
+1. Update `Module.md` and `Package.md` files
+
+These files will be displayed on the Ballerina Central package landing page. Ensure they introduce and explain the connector package clearly and comprehensively. Each file should include the following sections:
+
+- **Overview**: Provide a concise introduction to the connector, explaining its purpose and key features.
+- **Setup**: Offer step-by-step instructions on configuring the connector and any necessary prerequisites, such as API keys or environment setup.
+- **Quickstart**: Include a simple, clear example that helps users start using the connector immediately.
+- **Examples**: Link to additional use cases, providing context on how the connector can be used in different scenarios.
+
+> For reference, check the [Twitter connector documentation](https://github.com/ballerina-platform/module-ballerinax-twitter/blob/main/ballerina/Module.md).
+
+2. Update the `README.md` file
+
+This file will be displayed on the GitHub repository landing page. Therefore, it should include the same information as `Module.md` with a few additional sections such as `Building from Source`, `Contributing`, `License`, etc.
+
+> For reference, check the [Twitter connector README](https://github.com/ballerina-platform/module-ballerinax-twitter/blob/main/README.md).
+
+3. Write Example Use Cases (Optional)
```

Comment:
Should be in sentence case.

Reply:
Addressed in https://github.com/ballerina-platform/ballerina-dev-website/commit/7f3e76648db2831c1d956d733e403ad1999cc40d